salt/transport/local.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from salt.transport.client import ReqChannel
+
+log = logging.getLogger(__name__)
+
+
+class LocalChannel(ReqChannel):
+    '''
+    Local channel for testing purposes
+    '''
+    def __init__(self, opts, **kwargs):
+        self.opts = opts
+        self.kwargs = kwargs
+        self.tries = 0
+
+    def send(self, load, tries=3, timeout=60):
+
+        if self.tries == 0:
+            log.debug('LocalChannel load: {0}').format(load)
+            #data = json.loads(load)
+            #{'path': 'apt-cacher-ng/map.jinja', 'saltenv': 'base', 'cmd': '_serve_file', 'loc': 0}
+            #f = open(data['path'])
+            f = open(load['path'])
```

Comment:
Yea, I can change that over.. I didn't actually write this one, I just moved it ;) I'm not sure why its needed as you can easily test the channels without using this one... I think its legacy so I'd like to remove it at some point ;)